traffic_ops/traffic_ops_golang/deliveryservice/deliveryservices_required_capabilities_test.go

```diff
@@ -78,7 +78,7 @@ func TestCreateDeliveryServicesRequiredCapability(t *testing.T) {
 	mockTenantID(t, mock, 1)
 
 	arrayRows := sqlmock.NewRows([]string{"array"})
-	mock.ExpectQuery("SELECT server FROM deliveryservice_server").WillReturnRows(arrayRows)
+	mock.ExpectQuery("SELECT ds.server FROM deliveryservice_server").WillReturnRows(arrayRows)
```

Comment:
no real comment here, just curious why this needed to be done? should it be documented?

Reply:
I changed the query and the mock library does a regex on the query to mock the return